src/Microsoft.ML.Api/DataViewConstructionUtils.cs

```diff
@@ -946,7 +946,15 @@ public override ValueGetter<TDst> GetGetter<TDst>()
             throw Contracts.ExceptNotImpl("Type '{0}' is not yet supported.", typeT.FullName);
         }
 
-        internal override Delegate GetGetterDelegate() => Utils.MarshalInvoke(GetGetter<int>, MetadataType.RawType);
+        private Delegate GetGetterCore<TDst>()
```

Comment:
>GetGetterCore [](start = 25, length = 13)

I'm a bit confused. 
Why do you need this function if all it does is returns GetGetter?
Why you can't call GetGetter directly instead of calling this GetGetterCore function?

Reply:
Can you at least add comment which would justify presence of this function, to protect it from deletion?

---
In reply to: [240750686](https://github.com/dotnet/machinelearning/pull/1836#discussion_r240750686) [](ancestors = 240750686)